marin/speedrun/speedrun.py

```diff
@@ -73,22 +116,31 @@ def estimate_flops_via_6nd(self) -> float:
                 self.train_config.train_batch_size * self.model_config.seq_len * self.train_config.num_train_steps
             )
 
-        return 6.0 * N * total_tokens
+        flops_per_token = self.model_config.flops_per_token(self.vocab_size)
 
-    def estimate_flops_before_speedrun(self) -> tuple[bool, str]:
+        flops_per_token *= 3  # fwd + bwd
 
-        # estimate model FLOPs as 6*N*D, and calculate estimated compute using this and (a reasonable estimate of) MFU
-        estimated_model_flops = self.estimate_flops_via_6nd()
+        estimated_model_flops = flops_per_token * total_tokens
 
         logger.info(
-            f"""The rough estimated compute (calculated as 6*N*D / (Estimate of MFU))
-            for your run is around {estimated_model_flops/0.5:.2e} FLOPs assuming an MFU of 0.5.
-            This is calculated based on plausible MFU values and can be used as a rough estimate to
-            guide your config/training setup."""
+            f"""The rough estimated compute (calculated as (total model FLOPs / Assumed MFU))
```

Comment:
Can we make bullets for the two MFU cases on separate lines so it's easier to read?